Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM node:14-alpine
+FROM python:2-alpine
```

Comment:
This should be Python 3, we don't want to support Python 2 anymore

Reply:
Please see #1311

Reply:
Yeah, I already commented there as well. Python 3 is supported currently, Python 2 is not required and might be documented as not support at all even.

Reply:
This is not true AFAICT. We are using a version of `node-sass` that uses a version of `node-gyp` that doesn't support Python 3. I finally hit the same issue in `bourbon-neat` and gave up, since Wyrm didn't allow an upgrade. See #1311

Reply:
The theme has been building on Python 3 for some time. This PR has passing builds for all Pythons: https://github.com/readthedocs/sphinx_rtd_theme/pull/1316

Reply:
I haven't found an explanation for that TBH, but I'm sure it's good :) When I change the Dockerfile to use ` FROM python:3-alpine`, I start getting errors that I never resolved, but I really did try to do it without touching our node stack. The "internet" seems to have similar issues.

https://github.com/sass/node-sass/issues/2877
https://github.com/nodejs/node-gyp/issues/1687

Reply:
Odd, the errors don't necessarily surprise me, but I certainly can't offer an explanation. `node-gyp` apparently has python 3 support since 4.0.0 -- I hit this issue back in #771 -- https://github.com/readthedocs/sphinx_rtd_theme/pull/771#issuecomment-512689735

Reply:
We figured this out, seems Alpine is somehow using Node 18 and also cannot find the `/usr/bin/python3` binary (or Alpine's python package only installs a `/usr/bin/python` binary